02 ChangeName/readme.md

```diff
@@ -188,15 +188,15 @@ export const NameEditContainer = connect(
 ```diff
```

Comment:
If it's new file we can replace `diff` by `javascript` so Github can render styles. What do you think?

Reply:
Good idea. Done.